### ISSUE TITLE
Code inside <c></c> is not highlighted in XML Doc

### ISSUE THREAD

**Issue by vasily-kirichenko** [dotnet/fsharp#2670]
![image](https://cloud.githubusercontent.com/assets/873919/24146419/b31c3676-0e46-11e7-8068-4148316f5d68.png)

No go to definition links either. 

/cc @majocha

**Comment by majocha** [comment]
It was like this before.
It is not tagged as type so it doesn't get style or links.
I touched as little as possible in `niceprint.fs` et al. I'll try to look into this but the code in there is quite intimidating :)

**Comment by vasily-kirichenko** [comment]
I think @vladima rewrote from scratch the part that generates layout for XML Doc comments, so it seems easy to add parsing <c> tags there.

**Comment by vasily-kirichenko** [comment]
@majocha like add some more stuff here https://github.com/Microsoft/visualfsharp/blob/master/vsintegration/src/FSharp.LanguageService/XmlDocumentation.fs#L125 :)

**Comment by majocha** [comment]
@vasily-kirichenko indeed, styling as type should be easy. Resolving the text as type for browsing might be tricky, but I think still can be done.

**Comment by majocha** [comment]
Adding
```F#
            | "c" ->
                WriteTypeName collector el.Value
```
does the job of tagging, assuming the text in `<c>` is a type. There is no check for this at the moment.

**Comment by smoothdeveloper** [comment]
`c` isn't meant for type names but more generally, just code: https://docs.microsoft.com/en-us/dotnet/articles/fsharp/language-reference/xml-documentation

AFAIU, it is only `cref` attributes that we could interpret as pointing to a type or member or parameter name.

**Comment by dsyme** [comment]
Reopening as this is the canonical issue for this feature request right now
